.github/workflows/create-release.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       security-events: write
       contents: read
       actions: read
-    uses: dBildungsplattform/dbp-github-workflows/.github/workflows/release-image-and-helm-chart.yaml@5
+    uses: dBildungsplattform/dbp-github-workflows/.github/workflows/release-image-and-helm-chart.yaml@DBP-679-create-release-gha
```

Comment:
Why do you use the branch name as a tag instead of the SemVer?

Reply:
still debugging, will get changed back